pkg/appconsts/versioned_consts.go

```diff
@@ -1,12 +1,12 @@
 package appconsts
 
 import (
+	testground "github.com/celestiaorg/celestia-app/pkg/appconsts/testground"
```

Comment:
can the import alias be dropped?

```suggestion
	"github.com/celestiaorg/celestia-app/pkg/appconsts/testground"
```

Reply:
yeah we can good catch